lerna.json

```diff
@@ -0,0 +1,9 @@
+{
+  "lerna": "2.0.0",
+  "packages": [
+    "packages/*",
+    "."
```

Comment:
root directory could be removed and root's `package.json` marked as `"private": true` as this root "package" seems to be deprecated anyway